uint64 offset_ns	# time offset between companion system and PX4, in nanoseconds
